apps/partners-reference/src/applications/PaperApplicationForm/PaperApplicationForm.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useContext } from "react"
+import { useRouter } from "next/router"
+import {
+  ApiClientContext,
+  t,
+  Tag,
+  AppearanceSizeType,
+  Form,
+  AlertBox,
+  AlertTypes,
+} from "@bloom-housing/ui-components"
+import { useForm, FormProvider } from "react-hook-form"
+import { HouseholdMember } from "@bloom-housing/core"
+import { formatApplicationData } from "../../../lib/formatApplicationData"
+
+import { FormApplicationData } from "./sections/FormApplicationData"
+import { FormPrimaryApplicant } from "./sections/FormPrimaryApplicant"
+import { FormAlternateContact } from "./sections/FormAlternateContact"
+import { FormHouseholdMembers } from "./sections/FormHouseholdMembers"
+import { FormHouseholdDetails } from "./sections/FormHouseholdDetails"
+import { FormPreferences } from "./sections/FormPreferences"
+import { FormHouseholdIncome } from "./sections/FormHouseholdIncome"
+import { FormDemographics } from "./sections/FormDemographics"
+import { FormTerms } from "./sections/FormTerms"
+
+import { FormAside } from "./FormAside"
+
+const ApplicationForm = () => {
+  const router = useRouter()
+  const listingId = router.query.id as string
+  const { applicationsService } = useContext(ApiClientContext)
+
+  const [alert, setAlert] = useState<AlertTypes | null>(null)
+  const [householdMembers, setHouseholdMembers] = useState<HouseholdMember[]>([])
+
+  const formMethods = useForm()
```

Comment:
Is it https://github.com/bloom-housing/bloom/issues/847 applicable here?

Reply:
Now it is :) Good catch!